src/screen/react-nativeconfig.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    assets: ['./assets/Fonts']
+}
```

Comment:
delete this file
as this is not of any use. this way is used in react native app, not expo apps